### DISCUSSION TITLE
Displays a delete button when hovering over the component and provides a method for deletion

### DISCUSSION THREAD

**Discussion by luoqiz** [primefaces/community#1268] [PrimeVue] [answered]
![image](https://github.com/primefaces/community/assets/22161059/efa69d0c-0f9f-4d83-bc6f-f30a28b09b31)

**Comment by tugcekucukoglu** [comment, marked as answer]
You could use `#preview` slot and its `previewCallback` options. https://primevue.org/image/#template
